src/tribler-core/tribler_core/restapi/rest_manager.py

```diff
@@ -33,7 +34,13 @@ async def error_middleware(request, handler):
         if os.environ.get('TRIBLER_SHUTTING_DOWN', "FALSE") == "TRUE":
             raise Exception('Tribler is shutting down')
         response = await handler(request)
+    except web.HTTPMethodNotAllowed:
+        response = await handler(request)
```

Comment:
Why is this code here?

Reply:
please ignore everything on rest_manager.py i will revert it back to original when done, these are there only for debugger step in/out purposes.